specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json

```diff
@@ -224,6 +274,49 @@
         }
       },
       "description": "List of previewed features."
+    },
+    "Operation": {
+      "description": "Microsoft.Features operation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name: {provider}/{resource}/{operation}",
+          "type": "string"
+        },
+        "display": {
+          "description": "The object that represents the operation.",
+          "properties": {
+            "provider": {
+              "description": "Service provider: Microsoft.Features",
+              "type": "string"
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed: Profile, endpoint, etc.",
```

Comment:
Should this be an enum of specific values? Or is it arbitrary depending on the feature?